docs/WritingFirstTest-README.md

```diff
@@ -0,0 +1,94 @@
+# Writing the first test
+
+## Pre-requisites:
+
+        1. Intellij (install cucumber JVM plugin)
+        2. Java JDK 11 (JAVA_HOME Path Set)
+        3. Appium version (minimum 1.20.0)
+        4. Android studio (Android_Home Path Set)
+        5. GIT
+        6. Docker
+
+## Flow understanding
+
+    # Feature Files:    
+            These are the files that contains the tests in BDD format with the tags related to 
```

Comment:
This section needs to be improved.
Tests are written in the declarative style, depicting the business functionality. There should be no "UI actions" in the scenarios.
Depending on the platform your scenario is implemented for, you would need to add @android / @ios / @web @windows tags. You can add custom tags on the scenarios to help run tests independently, and also group them appropriately in the reports.